packages/perseus-editor/src/widgets/interactive-graph-editor.tsx

```diff
@@ -114,7 +114,7 @@ type Props = {
      * - grid: shows only the grid lines
      * - none: shows no markings
      */
-    markings: string; // TODO(jeremy)
+    markings: "graph" | "grid" | "none";
```

Comment:
We should probably have a pattern where we document them in `perseus-types.ts` and then reference those prop types using indexed accessors `markings: InteractiveGraphWidgetOptions["markings"]`.  The only place that doesn't work is when the widget defines a `transform` function whereby the widget options are not the same as the React component's Props.  But I think that's rare.

Reply:
@nedredmond For context, I added the prop defs here so that they would show up in storybook. Maybe we could switch around some stuff and add them to `types` later!